src/custom/state/lists/hooks/hooksMod.ts

```diff
@@ -1,6 +1,119 @@
+import { useMemo } from 'react'
 import { useSelector } from 'react-redux'
 import { AppState } from '../..'
+import { ChainId } from '@uniswap/sdk'
+import { useActiveWeb3React } from 'hooks'
+import { DEFAULT_NETWORK_FOR_LISTS, UNSUPPORTED_LIST_URLS } from 'constants/lists'
+import { TokenList } from '@uniswap/token-lists'
+import DEFAULT_TOKEN_LIST from '@uniswap/default-token-list'
+import { TokenAddressMap, listToTokenMap, combineMaps, EMPTY_LIST } from '@src/state/lists/hooks'
+import sortByListPriority from 'utils/listSort'
+import UNSUPPORTED_TOKEN_LIST from 'constants/tokenLists/uniswap-v2-unsupported.tokenlist.json'
 
-export function useFullLists() {
-  return useSelector<AppState, AppState['lists']>(state => state.lists)
+export * from '@src/state/lists/hooks'
+
+// MOD
+// This mod file adds { chainId } support to the hooks
+
+export function useAllLists(): {
+  readonly [url: string]: {
+    readonly current: TokenList | null
+    readonly pendingUpdate: TokenList | null
+    readonly loadingRequestId: string | null
+    readonly error: string | null
+  }
+} {
+  // MOD: adds { chainId } support to the hooks
+  const { chainId = DEFAULT_NETWORK_FOR_LISTS } = useActiveWeb3React()
```

Comment:
the main MOD is this line, and the comment below (repeats a few times in different functions here)